### DISCUSSION TITLE
Question For Admin Support

### DISCUSSION THREAD

**Discussion by AndrewM521** [J-Tanzanite/Little-Anti-Cheat#194] [Q&A] [answered]
Does this support alerting multiple admins if a player is suspected or detected as cheating?

If so, I don't see where I can add more admins to alert. I have people as admins through SourceMod with its built in admin groups, but it doesn't alert anyone except me.

**Comment by sapphonie** [comment, marked as answer]
```c
bool is_player_admin(int client)
{
	/* Todo: I don't know if this is correct. */
	return CheckCommandAccess(client, "", ADMFLAG_GENERIC | ADMFLAG_KICK | ADMFLAG_SLAY, true);
}
```

If clients have all those flags, they should see detections

**Reply by AndrewM521** [reply]
Perfect, thank you for the response. 

Also loving lilac so far, keep up the good work!
